src/test/ratings.spec.js

```diff
@@ -0,0 +1,54 @@
+import chai, { expect } from 'chai';
+import chaiHttp from 'chai-http';
+import { config } from 'dotenv';
+import app from '../index';
+
+config();
+chai.use(chaiHttp);
+chai.should();
+
+describe('Testing Rating of articles', () => {
```

Comment:
could you restructure your tests so that it truncates the table after each test to avoid dependencies